README_dev.md

```diff
@@ -0,0 +1,204 @@
+Building the DAGMC Cubit plugin from source
+===========================================
+
+This guide is intended for developers and maintainers of the Cubit plugin.
+For installation instructions see the [installation guide](https://github.com/svalinn/Cubit-plugin/tree/develop/README_dev.md).
+
+
+Prerequisites
+=============
+
+In order to build the plugin, you must have access to Cubit and the Cubit SDK.
+Additionally, the following system packages must be present on
+your computer:
+
+* EIGEN3
+* HDF5
+
+On Ubuntu, these packages can be obtained by running
+
+```bash
+sudo apt install libeigen3-dev libhdf5-dev
+```
+
+The following packages are not available from the package manager and must be
+built yourself:
+
+* [MOAB 5.3.0](https://bitbucket.org/fathomteam/moab/src/master/)
+* [DAGMC 3.2](https://github.com/svalinn/DAGMC)
+
+
+Notes on Build Instructions
+===========================
+
+A non-source directory build is recommended. These build instructions assume
+that the plugin build will take place in the `${HOME}/plugin-build` directory,
+and they assume that the Cubit-plugin repo has been cloned into
+`${HOME}/plugin-build/Cubit-plugin`.
+
+**Before building anything, ensure that the `LD_LIBRARY_PATH` environment
+variable is empty**. Ensure that it remains empty when running Cubit as well.
+
+```bash
+unset LD_LIBRARY_PATH
+```
+
+Build MOAB
+==========
+
+MOAB must be built with HDF5 enabled. On Ubuntu 18.04, HDF5 is located in the
+`/usr/lib/x86_64-linux-gnu/hdf5/serial` directory, but it may be located
+somewhere else on other flavors or versions of Linux. MOAB should be built with
+the Eigen matrix algebra library instead of LAPACK. The
+`_GLIBCXX_USE_CXX11_ABI=0` flag is required for compatibility with Cubit.
+
+The following commands show how to correctly build the MOAB dependency. If HDF5
+is located somewhere other than `/usr/lib/x86_64-linux-gnu/hdf5/serial`, then
+replace the directory with the correct one.
+
+```bash
+cd ${HOME}/plugin-build
+mkdir -pv moab/bld
+cd moab
+git clone https://bitbucket.org/fathomteam/moab -b Version5.1.0
```

Comment:
Let's upgrade this to 5.3.0

Reply:
maybe we should do this in the same time as we upgrade the scrips ?

Reply:
I've changed to clone the new 5.3.0 version in the README. It has a slightly different branch naming style